src/converters/fileldtype-converter.ts

```diff
@@ -17,14 +18,14 @@ const SIMPLE_VALUE_TYPES = [
     "NUMBER",
     "DATETIME",
     "TIME",
-    "RECORD_NUMBER",
     "DROP_DOWN",
     "LINK",
     "CALC",
     "RADIO_BUTTON",
 ];
 
 const SIMPLE_VALUE_IN_SAVED_RECORD = [
+    "RECORD_NUMBER",
```

Comment:
I know to move `RECORD_NUMBER` here is a big breaking change.
However, nobody will update and control. It's only depends on timing of save.
So I thought we could move `RECORD_NUMBER` .